release-process.md

```diff
@@ -0,0 +1,30 @@
+# Release Process
+
+This document outlines the steps involved in the release process for the Telemetry Exporter project.
+
+## Versioning
+
+The project follows [Semantic Versioning](https://semver.org/) for versioning.
+
+> Major version zero `(0.Y.Z)` is reserved for development, anything MAY change at any time. The public API is not stable.
+
+## Release Planning and Development
+
+The features that will go into the next release are reflected in the
+corresponding [milestone](https://github.com/nginxinc/telemetry-exporter/milestones). Refer to
+the [Issue Lifecycle](/ISSUE_LIFECYCLE.md) document for information on issue creation and assignment to releases.
+
+## Releasing a New Version
+
+1. Create an issue to define and track release-related activities. Choose a title that follows the
+   format `Release X.Y.Z`.
+2. Stop merging any new work into the main branch.
+3. Check the release draft under the [GitHub releases](https://github.com/nginxinc/telemetry-exporter/releases) page to ensure that everything is in order.
```

Comment:
Are we missing a step? What creates the release draft?

Reply:
every commit to main will update the draft